pages/proposals/language/English.json

```diff
@@ -6,7 +6,7 @@
     "background": {
       "header": "Welcome to the MozFest 2017 Call for Proposals",
       "body": [
-        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>.",
+        "MozFest sessions cover a wide range of topics, but share two important qualities: they are <em>interactive</em> and <em>inclusive</em>. We welcome workshops and exhibits, and often shy away from sessions that are exclusively lectures and slide presentations.",
```

Comment:
that kind of makes it all the more important to get the phrasing right though, no? As a strong force for good, we don't shy away, we lay down our preferences =)